pysparkle/pysparkle/data_quality/data_quality.py

```diff
@@ -0,0 +1,28 @@
+from pyspark.sql import SparkSession
+
+from pysparkle.adls_utils import set_env, set_spark_properties
+from pysparkle.data_quality.data_quality_utils import create_datasource_config, create_expectation_suite, \
+execute_validations
+
+def data_quality_main(DQ_conf):
+    dataset_name = DQ_conf["dataset_name"]
+    print("Running Data Quality processing...")
+    spark = SparkSession.builder.appName(
+        f'Data Quality {DQ_conf["container_name"]} {dataset_name}'
+    ).getOrCreate()
+
+    set_env()
+    set_spark_properties(spark)
+
+    table_name = f'{DQ_conf["container_name"]}.{dataset_name}'
+    dm_data = spark.read.table(table_name)
+    gx_dataframe = create_datasource_config(
```

Comment:
the name of this variable is quite misleading - it says it's a dataframe, while it's a context. Maybe rename it to gx_context or gx_config?